### DISCUSSION TITLE
Editing the style and size of the arrows in dimensions.

### DISCUSSION THREAD

**Discussion by hrjrkr** [hlorus/CAD_Sketcher#92] [Feature Requests]
**Motivation / Existing Problem**

**Description**

**Alternative Solutions**

**Mockup**
Ideally add a quick drawing to show how this would look.

**Comment by exegetor** [comment]
Size is done and dusted!  We now have arrow size control in addon preferences.

Style, however, may never happen.  CPU resources are limited and I'd only spend them on visuals in order to solve a problem like preventing cluttered, cramped drawings for example.
If anyone can make a case for arrow styles that make life better for the user, then I'll take another look!

**Reply by TheMakerTales** [reply]
what about the draw line width of the arrow?
